apps/src/code-studio/components/progress/ScriptOverviewHeader.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component, PropTypes } from 'react';
+import ReactDOM from 'react-dom';
+import { connect } from 'react-redux';
+import ProtectedStatefulDiv from '@cdo/apps/templates/ProtectedStatefulDiv';
+import PlcHeader from '@cdo/apps/code-studio/plc/header';
+
+/**
+ * This component takes some of the HAML generated content on the script overview
+ * page, and moves it under our React root. This is done so that we can have React
+ * content above and below this.
+ * Long term, instead of generating the DOM elements in haml, we should pass the
+ * client the data and have React generate the DOM. Doing so should not be super
+ * difficult in this case
+ */
+class ScriptOverviewHeader extends Component {
+  static propTypes = {
+    plcHeaderProps: PropTypes.shape({
+      unitName: PropTypes.string.isRequired,
+      courseViewPath: PropTypes.string.isRequired,
+    })
+  };
+
+  componentDidMount() {
+    $('#lesson').appendTo(ReactDOM.findDOMNode(this.protected));
+  }
+
+  render() {
+    const { plcHeaderProps } = this.props;
```

Comment:
Worth a quick test, even just `expect(mount(<ScriptOverviewHeader/>)).not.to.throw()`

Reply:
:thumbsup: I actually have a follow-on PR in the works, that has that test (and others)